src/main/java/org/humancellatlas/ingest/core/MetadataDocument.java

```diff
@@ -42,17 +47,30 @@ protected MetadataDocument(EntityType type,
     }
 
     public MetadataDocument addToSubmissionEnvelope(SubmissionEnvelope submissionEnvelope) {
-        this.submissionEnvelope = submissionEnvelope;
-
+        if(!this.isLinkedToOpenSubmissionEnvelope()){
+            this.validationState = ValidationState.DRAFT;
```

Comment:
Don't set state manually here.  Instead use `enactStateTransition(ValidationState.DRAFT)` which will throw an error is the state transition isn't an allowed one (adds some extra safety)